docs/admins/howto/new-hub.rst

```diff
@@ -25,139 +25,196 @@ The major reasons for making a new hub are:
 
 If your reason is something else, it probably needs some justification :)
 
+Prereqs
+=======
+Working installs of the following utilities:
+  - sops
+  - hubploy
+  - gcloud/kubectl
+  - `cookiecutter <https://github.com/audreyr/cookiecutter>`_
+
+Setting up a new hub
+====================
+
+Name the hub
+------------
+Choose the ``<hubname>`` (typically the course or department). This is permanent.
+
+Determine deployment needs
+--------------------------
+Before creating a new hub, have a discussion with the instructor about the system requirements,
+frequency of assignments and how much storage will be required for the course. Typically, there
```

Comment:
Maybe solicit app/tool info too, like whether the class will use Python, R, or Julia; or RStudio, Shiny, linux desktop; what critical libraries, etc.

Reply:
see https://github.com/berkeley-dsep-infra/datahub/pull/4094/commits/90151f26f32061ecb5dad4c53b2519377b44fd9b